New Operations manual front page (#2183)
neo4j/docs-operations: modules/ROOT/pages/index.adoc

[[operations-manual]]
= Neo4j Operations

> :page-layout: docs-ndl

Reviewer: I tried to build it locally, but it didn't render correctly. It complains with: "Page layout specified by page not found: docs-ndl (reverting to default layout)". Do you know what I can do to view it properly?

* `neo4j-admin server windows-service`

=== Database commands

[.icon]
image:icon-dbsettings.svg[]

[.list]
* xref:tools/neo4j-admin/consistency-checker.adoc[`neo4j-admin database check`]
* xref:tools/neo4j-admin/neo4j-admin-store-info.adoc[`neo4j-admin database info`]
* xref:tools/neo4j-admin/neo4j-admin-import.adoc[`neo4j-admin database import`]
* xref:tools/neo4j-admin/upload-to-aura.adoc[`neo4j-admin database upload`]
* xref:tools/neo4j-admin/migrate-database.adoc[`neo4j-admin database migrate`]
* xref:backup-restore/copy-database.adoc[`neo4j-admin database copy`]

=== Database backup commands

[.icon]
image:icon-import.svg[]

[.list]
* xref:backup-restore/online-backup.adoc[`neo4j-admin database backup`]
* xref:backup-restore/aggregate.adoc[`neo4j-admin backup aggregate`]
* xref:backup-restore/inspect.adoc[`neo4j-admin backup inspect`]
* xref:backup-restore/restore-backup.adoc[`neo4j-admin database restore`]
* xref:backup-restore/offline-backup.adoc[`neo4j-admin database dump`]
* xref:backup-restore/restore-dump.adoc[`neo4j-admin database load`]

[.next-steps]
== Licenses and disclaimers

[.link]

> link:https://neo4j.com/docs/license[Licenses and disclaimers]

Reviewer: Hey, @davidoliverSP2, this is what we've been talking about. Could you be able to check with legal if we need the licenses on each manual at all, and, if we need them, whether we could link to the licenses that apply to this manual directly or at least to a specific docs' page where only this license is explained? If I am a user, I would rather read the whole license instead of a company's summary of it, but if the legal think that this is needed, that's fine with me. I just think it should be easy for the user to see what licenses we comply with.

:console: 0

[.display.cards]

> == Install, deploy and operate Neo4j in a self-managed environment

Reviewer: Looks much better now.
However, I still think that having an intro sentence looks better than starting with "For more information...". If we have to remove something, then let's remove these two sentences:
"For information on upgrading and migrating Neo4j, see the link:https://neo4j.com/docs/upgrade-migration-guide/[Neo4j Upgrade and Migration] documentation.
For link:https://neo4j.com/aura/[Aura], the Neo4j fully managed cloud service, see the link:https://neo4j.com/docs/aura[Neo4j Aura] documentation."

Author: You mean keeping only the info about the current Neo4j version then?

Reviewer: No, I mean removing this heading: Install, deploy and operate Neo4j in a self-managed environment and having a sentence instead like I already suggested:

```suggestion
Neo4j Operations includes all the operational details and instructions for installing, deploying, and operating Neo4j in a self-managed environment. 

The latest version of Neo4j is *{neo4j-version-exact}*.
```

Reviewer: But who created this layout? Can't we amend it to suit our needs? Or we can have slightly different layouts for different needs/manuals?

Author: Sebastian from the design team created this, DocOps executes it. We can make amends when necessary for a functional reason, which I believe it's not the case here. We need to make sure we follow a standard and create consistency across all docsets.

For information on upgrading and migrating Neo4j, see the link:https://neo4j.com/docs/upgrade-migration-guide/[Neo4j Upgrade and Migration] documentation.
For link:https://neo4j.com/aura/[Aura], the Neo4j fully managed cloud service, see the link:https://neo4j.com/docs/aura[Neo4j Aura] documentation. 

> The latest version of Neo4j is *{neo4j-version-exact}*.

Reviewer: ```suggestion

The latest version of Neo4j is *{neo4j-version-exact}*.
```

> === Tutorials and how-to guides

Reviewer: ```suggestion
=== How-tos and tutorials
```

[.icon]
image:icon-library.svg[]

[.list]
* xref:tutorial/neo4j-admin-import.adoc[Neo4j-admin import]
* xref:clustering/setup/deploy.adoc[Deploy a basic cluster]
* xref:clustering/setup/analytics-cluster.adoc[Deploy an analytics cluster]
* xref:/clustering/setup/discovery.adoc[Move from discovery service v1 to v2]
* xref:/clustering/disaster-recovery.adoc[Disaster recovery guide]
* xref:tutorial/access-control.adoc[Fine-grained access control]
* xref:tutorial/tutorial-composite-database.adoc[Set up and use a composite database]
* xref:tutorial/tutorial-sso-configuration.adoc[Configuring Neo4j Single Sign-On (SSO)]
* xref:docker/docker-compose-standalone.adoc[Deploy Neo4j using Docker Compose]
* xref:tutorial/tutorial-clustering-docker.adoc[Deploy a Neo4j cluster in a Docker container]
* xref:kubernetes/quickstart-standalone/index.adoc[Deploy a standalone instance  using Helm charts]
* xref:kubernetes/quickstart-cluster/index.adoc[Deploy a cluster using Helm charts]
* xref:kubernetes/quickstart-analytics-cluster.adoc[Deploy a Neo4j cluster using Helm charts]

> === Reference

Reviewer: ```suggestion
=== References
```

[.icon]
image:icon-cheatsheet.svg[]

[.list]
* xref:/configuration/configuration-settings.adoc[Configuration settings]
* xref:procedures.adoc[Procedures]
* xref:/monitoring/metrics/index.adoc[Metrics]
* xref:/database-administration/syntax.adoc[Database management commands syntax]
* xref:/clustering/server-syntax.adoc[Server management commands syntax]
* xref:/docker/ref-settings.adoc[Docker configuration settings conversion table]
* xref:/configuration/ports.adoc[Ports in Neo4j]
* xref:/tools/neo4j-admin/index.adoc[Neo4j Admin and Neo4j CLI]
* xref:changes-deprecations-removals.adoc[Changes, deprecations, and removals]

> === Cypher admin commands

Reviewer: ```suggestion
=== Cypher administration commands
```

[.icon]
image:icon-developer.svg[]

[.list]
* xref:database-administration/standard-databases/naming-databases.adoc[Manage standard databases]
* xref:database-administration/aliases/naming-aliases.adoc[Manage database aliases]
* xref:database-administration/composite-databases/concepts.adoc[Manage composite databases]
* xref:clustering/servers.adoc[Manage servers in a cluster]
* xref:clustering/databases.adoc[Manage databases in a cluster]
* xref:authentication-authorization/manage-users.adoc[Manage users]
* xref:authentication-authorization/manage-roles.adoc[Manage roles]
* xref:authentication-authorization/manage-privileges.adoc[Manage privileges]
* xref:authentication-authorization/built-in-roles.adoc[Built-in roles and privileges]
* xref:authentication-authorization/immutable-roles-privileges.adoc[Manage immutable roles and privileges]

[.widget.lists]

> == Tailored for you

Author: This doesn't show

> == Tailored for you

Author: It's just how the component works. It doesn't show on the page, you can see it in the Surge preview

> == Tailored for you

Reviewer: Ok, but still, "Tailored for you" doesn't make any sense here. If you can rename it to == Neo4j Admin commands.

> === Neo4j Admin dbms commands

Reviewer: ```suggestion
=== DBMS commands
```

[.icon]
image:icon-admin.svg[]

[.list]
* xref:configuration/set-initial-password.adoc[`neo4j-admin dbms set-initial-password`]
* `neo4j-admin dbms set-default-admin`
* `neo4j-admin dbms unbind-system-db`

> === Neo4j Admin server commands

Reviewer: ```suggestion
=== Server commands
```

[.icon]
image:icon-server.svg[]

[.list]
* `neo4j-admin server console`
* `neo4j-admin server get-id`
* `neo4j-admin server license`
* xref:tools/neo4j-admin/neo4j-admin-memrec.adoc[`neo4j-admin server memory-recommendation`]
* xref:tools/neo4j-admin/migrate-configuration.adoc[`neo4j-admin server migrate-configuration`]
* xref:tools/neo4j-admin/neo4j-admin-report.adoc[`neo4j-admin server report`]
* `neo4j-admin server restart`
* `neo4j-admin server start`
* `neo4j-admin server status`
* `neo4j-admin server stop`
* xref:tools/neo4j-admin/unbind.adoc[`neo4j-admin server unbind`]
* xref:/tools/neo4j-admin/validate-config.adoc[`neo4j-admin server validate-config`]
* `neo4j-admin server windows-service`

> === Neo4j Admin database commands

Reviewer: ```suggestion
=== Database commands
```

[.icon]
image:icon-dbsettings.svg[]

[.list]
* xref:tools/neo4j-admin/consistency-checker.adoc[`neo4j-admin database check`]
* xref:tools/neo4j-admin/neo4j-admin-store-info.adoc[`neo4j-admin database info`]
* xref:tools/neo4j-admin/neo4j-admin-import.adoc[`neo4j-admin database import`]
* xref:tools/neo4j-admin/upload-to-aura.adoc[`neo4j-admin database upload`]
* xref:tools/neo4j-admin/migrate-database.adoc[`neo4j-admin database migrate`]
* xref:backup-restore/copy-database.adoc[`neo4j-admin database copy`]

> === Neo4j Admin database backup commands

Reviewer: ```suggestion
=== Database backup commands
```

[.icon]
image:icon-import.svg[]

[.list]
* xref:backup-restore/online-backup.adoc[`neo4j-admin database backup`]
* xref:backup-restore/aggregate.adoc[`neo4j-admin backup aggregate`]
* xref:backup-restore/inspect.adoc[`neo4j-admin backup inspect`]
* xref:backup-restore/restore-backup.adoc[`neo4j-admin database restore`]
* xref:backup-restore/offline-backup.adoc[`neo4j-admin database dump`]
* xref:backup-restore/restore-dump.adoc[`neo4j-admin database load`]

> [.next-steps]

Reviewer: Not sure if we need that. What does it mean?

Author: Just CSS for the component to work :)

Reviewer: Can we just have a sentence with no bullet or heading?

Author: No, that's how it is in all subpages.

> == Licenses and disclaimers

Reviewer: ```suggestion
```

[.link]

> * link:https://neo4j.com/docs/license[Licenses and disclaimers]

Reviewer: ```suggestion
Documentation license: link:https://neo4j.com/docs/reference/license/#_creative_commons[Creative Commons 4.0]
```
I am not sure if that will work, but my idea is to remove the heading and add the exact link.

Author: The standard for all pages is this one, we need to keep the whole link for legal reasons

Reviewer: Ok, I'll speak with David about it and let you know.

Reviewer: Hey, @davidoliverSP2, this is what we've been talking about. Could you be able to check with legal if we need the licenses on each manual at all, and, if we need them, whether we could link to the licenses that apply to this manual directly or at least to a specific docs' page where only this license is explained? If I am a user, I would rather read the whole license instead of a company's summary of it, but if the legal think that this is needed, that's fine with me. I just think it should be easy for the user to see what licenses we comply with.

:console: 0

[.display.cards.selectable]

== Manage and monitor your deployments

Find here details and instructions for installing and deploying Neo4j on-premise and in the cloud.
Follow tutorials and find administrative commands to help you manage and monitor your projects.

> === Installation

Author: I added the card for installation because it's the most visited section, and the links to Aura and the Upgrade and migrate because the original front page had links to it as a consequence of being other administrative options, but we can remove them all, if it makes more sense.

With regards to the Neo4j version, we can add that to the description in line 22. We don't need to link to the Deployment center though, only if we decide to keep the cards and then maybe replace the installation one with that.

[.icon]
image:icon-import.svg[]

[.description]
Install Neo4j in different deployment contexts, such as Linux, macOS, and Windows.

[.link]
xref:installation/index.adoc[]

[.cards]
=== Upgrade and migration

[.icon]
image:ROOT:icon-upgrade.svg[]

[.description]
See information on how to upgrade or migrate self-managed Neo4j deployments.

[.link]

> xref:installation/index.adoc[]

Author: This link is wrong because linking to another docset (using link instead of xref) is causing a problem in the UI, so I will fix that later if you agree with the suggestion of the content.

=== Aura instances

[.icon]
image:ROOT:icon-aura.svg[]

[.description]
See administrative options for fully managed cloud deployment.

[.link]

> xref:installation/index.adoc[]

Author: This link is wrong because linking to another docset (using link instead of xref) is causing a problem in the UI, so I will fix that later if you agree with the suggestion of the content.

[.widget.lists]
== Tailored for you

[.caption]
Keep exploring

=== Neo4j admin commands

[.icon]
image:icon-admin.svg[]

[.list]
* xref:/tools/neo4j-admin/consistency-checker.adoc[`database check`]
* xref:/tools/neo4j-admin/neo4j-admin-report.adoc[`server report`]
* xref:/tools/neo4j-admin/neo4j-admin-store-info.adoc[`database info`]
* xref:/tools/neo4j-admin/neo4j-admin-memrec/[`server memory-recommendation`]
* xref:/tools/neo4j-admin/neo4j-admin-import/[`database import`]
* xref:/tools/neo4j-admin/unbind.adoc[`server unbind`]
* xref:/tools/neo4j-admin/upload-to-aura.adoc[`database upload`]
* xref:/tools/neo4j-admin/migrate-database.adoc[`database migrate`]
* xref:/tools/neo4j-admin/migrate-configuration.adoc[`nserver migrate-configuration`]
* xref:/tools/neo4j-admin/validate-config.adoc[`server validate-config`]
* xref:/backup-restore/online-backup.adoc[`database backup`]

> * xref:/backup-restore/aggregate.adoc[`database aggregate`]

Reviewer 2: ```suggestion
* xref:/backup-restore/aggregate.adoc[`backup aggregate`]
```

* xref:authentication-authorization/manage-users.adoc[Manage users]
* xref:authentication-authorization/manage-roles.adoc[Manage roles]
* xref:authentication-authorization/manage-privileges.adoc[Manage privileges]
* xref:authentication-authorization/built-in-roles.adoc[Built-in roles and privileges]
* xref:authentication-authorization/immutable-roles-privileges.adoc[Manage immutable roles and privileges]

=== Reference

[.icon]
image:icon-cheatsheet.svg[]

[.list]
* xref:/configuration/configuration-settings.adoc[Configuration settings]
* xref:procedures.adoc[Procedures]
* xref:/monitoring/metrics/index.adoc[Metrics]
* xref:/database-administration/syntax.adoc[Database management commands syntax]
* xref:/clustering/server-syntax.adoc[Server management commands syntax]
* xref:/docker/ref-settings.adoc[Docker configuration settings conversion table]
* xref:/configuration/ports.adoc[Ports in Neo4j]
* xref:/tools/neo4j-admin/index.adoc[Neo4j Admin and Neo4j CLI]
* xref:changes-deprecations-removals.adoc[Changes, deprecations, and removals]

=== Cloud object storage integration (COSI)

[.icon]
image:icon-cloudstorage.svg[]

[.list]

> * Restore from the cloud

Author: Links needed

Reviewer 2: I'll go through the list and provide links. But there was a question from @renetapopova if we need this column at all.

* Restore backup
* Load dump
* Import from the cloud
* Create from the cloud resource
* Seed from URI
* Upload a backup to the cloud

[.widget.highlights]
== Tutorials & How-to guides

[.icon]

> image:ga-home.svg[]

Author: I can't see your screenshot, unfortunately. Can you send it to me over Slack so I understand what you mean?

:console: 0

[.display.cards.selectable]

> == Manage and monitor your deployments

Reviewer: ```suggestion
```

> Find here details and instructions for installing and deploying Neo4j on-premises and in the cloud.

Reviewer: ```suggestion
```

> Follow tutorials and find administrative commands to help you manage and monitor your projects.

Reviewer: ```suggestion
```

> The latest version of Neo4j is *{neo4j-version-exact}*.

Reviewer: ```suggestion
The Operations manual includes all the operational details and instructions for installing, deploying, and operating Neo4j in a self-managed environment. +
For all information on upgrading and migrating Neo4j, see the [Neo4j Upgrade and Migration Guide](https://neo4j.com/docs/upgrade-migration-guide/current/). +
For more information on [Aura](https://neo4j.com/aura/), the Neo4j fully managed cloud service, see the [Neo4j Aura Manual](https://neo4j.com/docs/aura). 

The latest version of Neo4j is *{neo4j-version-exact}*.
```

Author: I made another suggestion, please check the new version

> [.widget.lists]

Reviewer: ```suggestion
```

Author: This is a css class for the component to work.

> == Tailored for you

Reviewer: ```suggestion
```

Author: This doesn't show in the page. It just needs to be there for the component to work properly.

Reviewer: Ok. That's fine. We need to revert the other heading then.
